esp-alloc/src/lib.rs

```diff
@@ -61,35 +67,81 @@ use core::{
 };
 
 use critical_section::Mutex;
+use enumset::{EnumSet, EnumSetType};
 use linked_list_allocator::Heap;
 
+/// The global allocator instance
+#[global_allocator]
+pub static INSTANCE: EspHeap = EspHeap::empty();
```

Comment:
Maybe just calling it `HEAP` is a bit more clear, `esp_alloc::HEAP` is maybe a bit more intuitive than `esp_allow::INSTANCE` from a user's perspective?